extensions/httpserver/libhttpserver.m

```diff
@@ -583,7 +583,12 @@ static int httpserver_start(lua_State *L) {
     LuaSkin *skin = [LuaSkin sharedWithState:L];
     HSHTTPServer *server = getUserData(L, 1);
 
-    if (server.fn == LUA_NOREF) {
+    if (server.wsCallback != LUA_NOREF) {
+        NSError *error = nil;
+        if (![server start:&error]) {
+            [skin logError:[NSString stringWithFormat:@"hs.httpserver:start() Unable to start object: %@", error]];
+        }
+    else if (server.fn == LUA_NOREF) {
```

Comment:
Rather than duplicating the `[server start:]` call, how about we just check if both `server.wsCallback` and `server.fn` are `LUA_NOREF` and use that to throw the "no callback set" error?